alb.tf

```diff
@@ -19,6 +19,7 @@ resource "aws_alb_target_group" "main" {
   proxy_protocol_v2                  = lookup(var.target_groups[count.index], "proxy_protocol_v2", false)
   lambda_multi_value_headers_enabled = lookup(var.target_groups[count.index], "lambda_multi_value_headers_enabled", false)
   load_balancing_algorithm_type      = lookup(var.target_groups[count.index], "load_balancing_algorithm_type", null)
+  connection_termination	     = true
```

Comment:
@thisismana why not make this configurable like the other attributes in `aws_lb_target_group`? to be backwards compatible, the [default](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#connection_termination) would need to be `false` - do we need to switch this to `true`?